### ISSUE TITLE
linenumber can't load correctly at line 2008

### ISSUE THREAD

**Issue by Moosems** [not-nef/txt2#34]
We need to make the linenumbers dynamically resize
<img width="91" alt="Screen Shot 2022-07-17 at 12 10 44 PM" src="https://user-images.githubusercontent.com/95927277/179416945-bdd36507-e7f8-4244-b490-0884913016f2.png">

**Comment by not-nef** [comment]
yea

**Comment by not-nef** [comment]
you bug huntin or what

**Comment by Moosems** [comment]
Yeah kinda. When I work on something I try to make it perfect so I do bug hunting sessions to find things I can fix

**Comment by not-nef** [comment]
huh? i didnt even change the code:

![image](https://user-images.githubusercontent.com/83908932/179790418-c2854294-0fd5-4f7b-acab-f446c9b4c43d.png)

**Comment by not-nef** [comment]
I made it increase the width when it passes 2000 lines anyway
